debian/tenstorrent-tools.install

```diff
@@ -1,5 +1,5 @@
 hugepages-setup.sh /opt/tenstorrent/bin
-tenstorrent-hugepages.service /lib/systemd/system/
-dev-hugepages\\x2d1G.mount /lib/systemd/system/
+debian/tenstorrent-hugepages.service /lib/systemd/system/
```

Comment:
these files aren't in the debian/ directory, and changing this will mess up the .deb file creation itself (and if it works it won't have these files)

Reply:
I've moved these files to the debian dir in the PR, you should be able to see those changes also in the PR (github lists them as renames)

Reply:
they should not be in the debian directory as we'll need those when we generate non-debs as well, so it doesn't make sense for those to be there

Reply:
`debhelper` seems to generate the `postinst` and `postrm` steps (the part where the services get started and cleaned up) properly only if these files are in the debian directory. I download the deb files of a couple of packages (sshd, mariadb) and they all have the systemd service files within the debian dir in them. So I thought that this was the right place to keep these files.

Since you are saying that we will need them for non-debian distros as well, do you have a suggestion for a way to accomplish this? Creating a copy or copying the file into the debian dir during the build process or something better?